Data validation error: Problem validating JSON document against schema on JRUBY

I am using apm-agent with jruby. If you don't have `JRUBY_VERSION` set then this will lead to following error,
```
2018-07-31T09:43:48.883Z    ERROR    [request]    beater/handlers.go:414    error handling request    {"request_id": "0f2d5500-682d-4c8e-a8c8-0f3a2985819f", "method": "POST", "URL": "http://elasticsearch.com:8200/v1/transactions", "content_length": 3822, "remote_address": "x.x.x.x", "user-agent": "elastic-apm/ruby 1.0.1", "response_code": 400, "error": "data validation error: Problem validating JSON document against schema: I[#] S[#] doesn't validate with \"transaction#\"\n  I[#/service/runtime/version] S[#/properties/service/properties/runtime/properties/version/type] expected string, but got null"}
```
Error is reported for service runtime version. I had a look at the code and found this line,
https://github.com/elastic/apm-agent-ruby/blob/f19944fcb3087a98102c54c10e6781e9b2b57188/lib/elastic_apm/service_info.rb#L50

I think we should add it to documentation to set `JRUBY_VERSION` before starting apm-agent-ruby

--- a/lib/elastic_apm/service_info.rb
+++ b/lib/elastic_apm/service_info.rb
@@ -45,9 +45,9 @@ module ElasticAPM
     def runtime
       case RUBY_ENGINE
       when 'ruby'
-        { name: RUBY_ENGINE, version: RUBY_VERSION }
+        { name: RUBY_ENGINE, version: RUBY_VERSION || RUBY_ENGINE_VERSION }
       when 'jruby'
-        { name: RUBY_ENGINE, version: ENV['JRUBY_VERSION'] }
+        { name: RUBY_ENGINE, version: JRUBY_VERSION || RUBY_ENGINE_VERSION }
       end
     end
   end

